### DISCUSSION TITLE
What is the advantage of having Chrome DOM when you already have Wget?

### DISCUSSION THREAD

**Discussion by melyux** [ArchiveBox/ArchiveBox#1176] [Q&A] [answered]
Curious about this. Seems like the Chrome DOM snapshot is inferior to Wget in every way (no offline images being the main issue). What are the benefits of keeping the DOM snapshots?

**Comment by pirate** [comment, marked as answer]
redundancy, there are plenty of sites that only work with one or the other but not both
